eahub/config/settings.py

```diff
@@ -195,8 +197,10 @@
 CRISPY_TEMPLATE_PACK = "bootstrap3"
 
 # Django PWNED Passwords
+PWNED_VALIDATOR_ERROR = forms.ValidationError(mark_safe("Your password was determined to have been involved in a major security breach in the <a target='_blank' href='https://haveibeenpwned.com/passwords'>past</a>."))
```

Comment:
Do you need the `ValidationError` here? I think just `mark_safe` works.

Reply:
Done. Feel free to make any other changes you feel are necessary.

Reply:
I think this commit didn't get pushed.